tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/cli/dockerCli/core/DockerContext.ts

```diff
@@ -0,0 +1,102 @@
+import { Logger } from "winston";
+import { DockerCliConfig } from "../schema/dockerCliConfig";
```

Comment:
Please enable the tool to scan and fix the code style. As I know, one basic rule is to sort the import.

Reply:
There is another task to enable the rule. I don't enable it now because zhouzheng also has a PR, which contains a lot changes. I don't want too many conflicts.